### PR TITLE
[BEAM-2977] Improve unbounded prose in wordcount example

### PR DESCRIPTION
**Pull request by melap**

### CODE REVIEW

**Comment by melap** [comment]
Staged: http://apache-beam-website-pull-requests.storage.googleapis.com/377/get-started/wordcount-example/index.html#unbounded-and-bounded-data-sets

File: src/get-started/wordcount-example.md [hunk 1]
```diff
@@ -773,12 +773,21 @@ To view the full code in Java, see
 
 > **Note:** WindowedWordCount is not yet available for the Python SDK.
 
-### Unbounded and bounded pipeline input modes
+### Unbounded and bounded data sets
 
 Beam allows you to create a single pipeline that can handle both bounded and
 unbounded types of input. If your input has a fixed number of elements, it's
```
Comment (tgroh) [inline src/get-started/wordcount-example.md]: `bounded and unbounded input` or `bounded and unbounded input data sets`, for consistency

File: src/get-started/wordcount-example.md [hunk 2]
```diff
@@ -773,12 +773,21 @@ To view the full code in Java, see
...
-considered a 'bounded' data set. If your input is continuously updating, then
-it's considered 'unbounded' and you must use a runner that supports streaming.
+considered a bounded data set. For bounded data sets, the question you ask is
```
Comment (tgroh) [inline src/get-started/wordcount-example.md]: "If your input has a fixed number of elements, it is a bounded data set, and all of the data can be processed together. If input data continuously arrives (such as...), it is an unbounded data set. An unbounded data set will only be complete up to a certain point, and must be processed with..."

(roughly)

**Comment by melap** [comment]
retest this please

**Comment by melap** [comment]
retest this please

**Comment by melap** [comment]
@lukecwik can you take a quick look at this? if it looks ok, I'll squash and then merge.

**Comment by lukecwik** [comment]
nit: you missed one reference to `data set` around `used in unit tests with small data sets.`

**Comment by melap** [comment]
good catch, will fix that, thanks!

**Comment by melap** [comment]
@asfgit merge

**Comment by asfgit** [comment]
Error: PR failed in verification; check the Jenkins job for more information.

**Comment by melap** [comment]
It's the dreaded gpg error... will try again later.

**Comment by melap** [comment]
@asfgit merge

**Comment by asfgit** [comment]
Error: PR failed in verification; check the Jenkins job for more information.

**Comment by melap** [comment]
@asfgit merge

**Pull request merged**